aws_backendless_chef_ha.yml

```diff
@@ -0,0 +1,1052 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: >
+  AWS HA Chef Server 2.0 - Backendless Chef - Ubuntu Xenial Based
+  "Backendless" Chef utilizing AWS as Backend for PostgreSQL and Elasticsearch
+  Sets up the following:
+    Backendless Items
+    * Elasticsearch 2.3 Cluster
+    * PostgreSQL 9.5.4
+
+Parameters:
+
+  ## Domain Configuration
+  HostedZone:
+    Type: String
+    Default: domain.com
+    Description: must match a route53 hosted domain/zone
+
+  SSLCertificateARN:
+    Type: String
+    Default: 'arn:aws:iam::'
```

Comment:
It's just an example so someone knows exactly what they need to look for. When I first started this I thought they meant just a cert name, but it needs the full arn path. Mainly just there as a reminding for what I'm providing. I can provide a comment to the page that explains it? Adding to much to the param description causes the pages to go sideways if using the console to launch.